tests/unit/remote/ssh/test_connection.py

```diff
@@ -64,3 +76,22 @@ def test_walk(tmp_path, ssh):
             paths.add(posixpath.join(root, entry))
 
     assert paths == expected
+
+@pytest.mark.skipif(platform.system() != "Darwin",
+                    reason="Only works on OSX (because of APFS)")
+def test_reflink(chdir_tmp, foo, ssh):
+    ssh.reflink("foo", "bar")
+    assert System.is_reflink(bar_path)
+
+@pytest.mark.skip(reason="MockSSH doesn't support symlinks")
```

Comment:
You can at least mock sftp.symlink and check that it is called.

Reply:
Also, you might be able to func test it same as we do in TestRemoteSSH in tests/func/test_data_sync.py.

Reply:
ready :+1: